libs/designsystem/src/lib/components/chart/options/donut.ts

```diff
@@ -76,7 +76,7 @@ export const DonutOptions: Options = {
     {
       name: '',
       type: 'pie',
-      data: this === undefined ? [] : this.data,
+      data: [],
```

Comment:
This doesn't seem like a safe refactoring... 🔥 
See this PR for an alternative solution: https://github.com/kirbydesign/designsystem/pull/1260